cli/module_loader.rs

```diff
@@ -78,12 +78,8 @@ impl CliModuleLoader {
     specifier: &ModuleSpecifier,
     maybe_referrer: Option<ModuleSpecifier>,
   ) -> Result<ModuleCodeSource, AnyError> {
-    if specifier.as_str() == "node:module" {
-      return Ok(ModuleCodeSource {
-        code: deno_runtime::deno_node::MODULE_ES_SHIM.to_string(),
-        found_url: specifier.to_owned(),
-        media_type: MediaType::JavaScript,
-      });
+    if specifier.scheme() == "node" {
+      unreachable!("Node built-in modules should be handled internally.");
```

Comment:
Maybe this should error instead? People could provide an incorrect `node:` specifier and I forget if that would be handled before here.

Reply:
That's the reason I put it 😅  - we should never get this far is specifier is invalid (because that module doesn't exist, so resolution should fail).